core/src/storage/device.py

```diff
@@ -36,6 +36,7 @@
 _EXPERIMENTAL_FEATURES     = const(0x15)  # bool (0x01 or empty)
 _HIDE_PASSPHRASE_FROM_HOST = const(0x16)  # bool (0x01 or empty)
 _BRIGHTNESS                = const(0x17)  # int
+_DISABLE_HAPTIC_FEEDBACK   = const(0x18)  # bool (0x01 or empty)
```

Comment:
The default is to have haptic feedback enabled - so when there is no value written is storage it needs to be interpreted as enabled too. 

Our boolean getter in storage/common returns `false` when no value is written, so false needs to be interpreted as enabled.

Reply:
Makes sense. Then I would have argued to rename everything else top match this, because indeed, we are adding an option to disable it, not to enable it. But this can also work...